diff --git a/src/audio_aac.c b/src/audio_aac.c
index 0667e50..db66b48 100644
--- a/src/audio_aac.c
+++ b/src/audio_aac.c
@@ -311,7 +311,7 @@ audio_profile_guess_aac_priv (AVCodecContext *ac, aac_object_type_t type)
     return AUDIO_PROFILE_INVALID;
 
   /* check for AAC variants codec */
-  if (ac->codec_id != CODEC_ID_AAC)
+  if (ac->codec_id != AV_CODEC_ID_AAC)
     return AUDIO_PROFILE_INVALID;
   
   switch (type)
diff --git a/src/audio_ac3.c b/src/audio_ac3.c
index 8d3c25f..6b2bba8 100644
--- a/src/audio_ac3.c
+++ b/src/audio_ac3.c
@@ -39,7 +39,7 @@ audio_profile_guess_ac3 (AVCodecContext *ac)
     return AUDIO_PROFILE_INVALID;
 
   /* check for AC3 codec */
-  if (ac->codec_id != CODEC_ID_AC3)
+  if (ac->codec_id != AV_CODEC_ID_AC3)
     return AUDIO_PROFILE_INVALID;
   
   /* supported channels: 1/0, 2/0, 3/0, 2/1, 3/1, 2/2, 3/2 */
diff --git a/src/audio_amr.c b/src/audio_amr.c
index 5347842..3fb6e8e 100644
--- a/src/audio_amr.c
+++ b/src/audio_amr.c
@@ -53,7 +53,7 @@ audio_is_valid_amr (AVCodecContext *ac)
   if (!ac)
     return 0;
 
-  if (ac->codec_id != CODEC_ID_AMR_NB)
+  if (ac->codec_id != AV_CODEC_ID_AMR_NB)
     return 0;
   
   /* only mono is supported */
@@ -89,7 +89,7 @@ audio_is_valid_amr_wb (AVCodecContext *ac)
   if (!ac)
     return 0;
 
-  if (ac->codec_id != CODEC_ID_AMR_WB)
+  if (ac->codec_id != AV_CODEC_ID_AMR_WB)
     return 0;
   
   /* valid sampling rates: 8, 16, 24, 32 and 48 kHz */
diff --git a/src/audio_atrac3.c b/src/audio_atrac3.c
index 22aace5..6ab2d5c 100644
--- a/src/audio_atrac3.c
+++ b/src/audio_atrac3.c
@@ -39,7 +39,7 @@ audio_profile_guess_atrac (AVCodecContext *ac)
     return AUDIO_PROFILE_INVALID;
 
 #if LIBAVCODEC_VERSION_INT >= ((51<<16)+(40<<8)+4)
-  if (ac->codec_id == CODEC_ID_ATRAC3)
+  if (ac->codec_id == AV_CODEC_ID_ATRAC3)
     return AUDIO_PROFILE_ATRAC;
 #endif
 
diff --git a/src/audio_g726.c b/src/audio_g726.c
index 98e0a17..71bc142 100644
--- a/src/audio_g726.c
+++ b/src/audio_g726.c
@@ -31,7 +31,7 @@ audio_profile_guess_g726 (AVCodecContext *ac)
   if (!ac)
     return AUDIO_PROFILE_INVALID;
 
-  if (ac->codec_id != CODEC_ID_ADPCM_G726)
+  if (ac->codec_id != AV_CODEC_ID_ADPCM_G726)
     return AUDIO_PROFILE_INVALID;
   
   if (ac->channels != 1)
diff --git a/src/audio_lpcm.c b/src/audio_lpcm.c
index 5a12193..98676fe 100644
--- a/src/audio_lpcm.c
+++ b/src/audio_lpcm.c
@@ -45,8 +45,8 @@ audio_profile_guess_lpcm (AVCodecContext *ac)
     return AUDIO_PROFILE_INVALID;
 
   /* check for 16-bit signed network-endian PCM codec  */
-  if (ac->codec_id != CODEC_ID_PCM_S16BE &&
-      ac->codec_id != CODEC_ID_PCM_S16LE)
+  if (ac->codec_id != AV_CODEC_ID_PCM_S16BE &&
+      ac->codec_id != AV_CODEC_ID_PCM_S16LE)
     return AUDIO_PROFILE_INVALID;
 
   /* supported channels: mono or stereo */
diff --git a/src/audio_mp2.c b/src/audio_mp2.c
index 2a58f78..4c65d87 100644
--- a/src/audio_mp2.c
+++ b/src/audio_mp2.c
@@ -31,7 +31,7 @@ audio_profile_guess_mp2 (AVCodecContext *ac)
   if (!ac)
     return AUDIO_PROFILE_INVALID;
 
-  if (ac->codec_id != CODEC_ID_MP2)
+  if (ac->codec_id != AV_CODEC_ID_MP2)
     return AUDIO_PROFILE_INVALID;
   
   if (ac->channels > 5)
diff --git a/src/audio_mp3.c b/src/audio_mp3.c
index 07cf50c..0c8636d 100644
--- a/src/audio_mp3.c
+++ b/src/audio_mp3.c
@@ -47,7 +47,7 @@ audio_is_valid_mp3_common (AVCodecContext *ac)
     return 0;
 
   /* check for MP3 codec */
-  if (ac->codec_id != CODEC_ID_MP3)
+  if (ac->codec_id != AV_CODEC_ID_MP3)
     return 0;
   
   /* only mono and stereo are supported */
diff --git a/src/audio_wma.c b/src/audio_wma.c
index 214b29e..33db1b6 100644
--- a/src/audio_wma.c
+++ b/src/audio_wma.c
@@ -55,7 +55,7 @@ audio_profile_guess_wma (AVCodecContext *ac)
     return ap;
 
   /* check for WMA codec */
-  if (ac->codec_id != CODEC_ID_WMAV1 && ac->codec_id != CODEC_ID_WMAV2)
+  if (ac->codec_id != AV_CODEC_ID_WMAV1 && ac->codec_id != AV_CODEC_ID_WMAV2)
     return ap;
 
   if (ac->sample_rate <= 48000)
diff --git a/src/av_mpeg1.c b/src/av_mpeg1.c
index 7f06d8d..219391d 100644
--- a/src/av_mpeg1.c
+++ b/src/av_mpeg1.c
@@ -42,7 +42,7 @@ probe_mpeg1 (AVFormatContext *ctx dlna_unused,
     return NULL;
   
   /* check for MPEG-1 video codec */
-  if (codecs->vc->codec_id != CODEC_ID_MPEG1VIDEO)
+  if (codecs->vc->codec_id != AV_CODEC_ID_MPEG1VIDEO)
     return NULL;
 
   /* video bitrate must be CBR at 1,151,929.1 bps */
@@ -72,7 +72,7 @@ probe_mpeg1 (AVFormatContext *ctx dlna_unused,
     return NULL;
 
   /* check for MPEG-1 Layer-2 audio codec */
-  if (codecs->ac->codec_id != CODEC_ID_MP2)
+  if (codecs->ac->codec_id != AV_CODEC_ID_MP2)
     return NULL;
   
   /* supported channels: stereo only */
diff --git a/src/av_mpeg2.c b/src/av_mpeg2.c
index 580e39a..72a6a62 100644
--- a/src/av_mpeg2.c
+++ b/src/av_mpeg2.c
@@ -453,7 +453,7 @@ probe_mpeg_ts (AVFormatContext *ctx,
   int i;
   
   /* check for MPEG-2 MP@LL profile */
-  if (codecs->ac->codec_id == CODEC_ID_AAC)
+  if (codecs->ac->codec_id == AV_CODEC_ID_AAC)
   {
     /* 352x288 only */
     if (codecs->vc->width != 352 && codecs->vc->height != 288)
@@ -535,7 +535,7 @@ probe_mpeg_ts (AVFormatContext *ctx,
   if (ctx->bit_rate > 19392700)
     return NULL;
 
-  if (codecs->ac->codec_id != CODEC_ID_AC3)
+  if (codecs->ac->codec_id != AV_CODEC_ID_AC3)
     return NULL;
 
   /* 48 KHz only */
@@ -615,7 +615,7 @@ probe_mpeg2 (AVFormatContext *ctx,
     return NULL;
   
   /* check for MPEG-2 video codec */
-  if (codecs->vc->codec_id != CODEC_ID_MPEG2VIDEO)
+  if (codecs->vc->codec_id != AV_CODEC_ID_MPEG2VIDEO)
     return NULL;
 
   switch (st)
diff --git a/src/av_mpeg4_part10.c b/src/av_mpeg4_part10.c
index f9c8ee5..3c28904 100644
--- a/src/av_mpeg4_part10.c
+++ b/src/av_mpeg4_part10.c
@@ -1221,7 +1221,7 @@ probe_avc (AVFormatContext *ctx,
     return NULL;
 
   /* check for H.264/AVC codec */
-  if (codecs->vc->codec_id != CODEC_ID_H264)
+  if (codecs->vc->codec_id != AV_CODEC_ID_H264)
     return NULL;
 
   /* check for a supported container */
diff --git a/src/av_mpeg4_part2.c b/src/av_mpeg4_part2.c
index 823e1b6..b631365 100644
--- a/src/av_mpeg4_part2.c
+++ b/src/av_mpeg4_part2.c
@@ -42,13 +42,13 @@ static const struct {
   int codec_id;
   mpeg4_vcodec_type_t type;
 } avf_vcodec_mapping[] = {
-  { CODEC_ID_H263,                      MPEG4_VCODEC_H263 },
-  { CODEC_ID_H263I,                     MPEG4_VCODEC_H263 },
-  { CODEC_ID_H263P,                     MPEG4_VCODEC_H263 },
-  { CODEC_ID_MPEG4,                     MPEG4_VCODEC_P2 },
-  { CODEC_ID_MSMPEG4V1,                 MPEG4_VCODEC_P2 },
-  { CODEC_ID_MSMPEG4V2,                 MPEG4_VCODEC_P2 },
-  { CODEC_ID_MSMPEG4V3,                 MPEG4_VCODEC_P2 },
+  { AV_CODEC_ID_H263,                      MPEG4_VCODEC_H263 },
+  { AV_CODEC_ID_H263I,                     MPEG4_VCODEC_H263 },
+  { AV_CODEC_ID_H263P,                     MPEG4_VCODEC_H263 },
+  { AV_CODEC_ID_MPEG4,                     MPEG4_VCODEC_P2 },
+  { AV_CODEC_ID_MSMPEG4V1,                 MPEG4_VCODEC_P2 },
+  { AV_CODEC_ID_MSMPEG4V2,                 MPEG4_VCODEC_P2 },
+  { AV_CODEC_ID_MSMPEG4V3,                 MPEG4_VCODEC_P2 },
   { -1, 0 }
 };
    
diff --git a/src/av_wmv9.c b/src/av_wmv9.c
index e8561a7..c746f9a 100644
--- a/src/av_wmv9.c
+++ b/src/av_wmv9.c
@@ -245,7 +245,7 @@ probe_wmv9 (AVFormatContext *ctx dlna_unused,
     return NULL;
 
   /* check for WMV3 (Simple and Main profiles) video codec */
-  if (codecs->vc->codec_id != CODEC_ID_WMV3)
+  if (codecs->vc->codec_id != AV_CODEC_ID_WMV3)
     return NULL;
 
   /* get video profile */
diff --git a/src/image_jpeg.c b/src/image_jpeg.c
index 135740c..a1866c8 100644
--- a/src/image_jpeg.c
+++ b/src/image_jpeg.c
@@ -92,10 +92,10 @@ probe_jpeg (AVFormatContext *ctx,
     return NULL;
 
   /* check for JPEG compliant codec */
-  if (codecs->vc->codec_id != CODEC_ID_MJPEG &&
-      codecs->vc->codec_id != CODEC_ID_MJPEGB &&
-      codecs->vc->codec_id != CODEC_ID_LJPEG &&
-      codecs->vc->codec_id != CODEC_ID_JPEGLS)
+  if (codecs->vc->codec_id != AV_CODEC_ID_MJPEG &&
+      codecs->vc->codec_id != AV_CODEC_ID_MJPEGB &&
+      codecs->vc->codec_id != AV_CODEC_ID_LJPEG &&
+      codecs->vc->codec_id != AV_CODEC_ID_JPEGLS)
     return NULL;
 
   for (i = 0; jpeg_profiles_mapping[i].profile; i++)
diff --git a/src/image_png.c b/src/image_png.c
index 1c2191a..b987d39 100644
--- a/src/image_png.c
+++ b/src/image_png.c
@@ -76,7 +76,7 @@ probe_png (AVFormatContext *ctx,
     return NULL;
   
   /* check for PNG compliant codec */
-  if (codecs->vc->codec_id != CODEC_ID_PNG)
+  if (codecs->vc->codec_id != AV_CODEC_ID_PNG)
     return NULL;
 
   for (i = 0; png_profiles_mapping[i].profile; i++)
