diff --git a/configure b/configure
index 62e53b0..3910549 100755
--- a/configure
+++ b/configure
@@ -7574,7 +7574,7 @@ if test "$alsa_prefix" != "" ; then
 	LDFLAGS="$LDFLAGS $ALSA_LIBS"
 fi
 
-ALSA_LIBS="$ALSA_LIBS -lasound -lm $LIBDL -lpthread"
+ALSA_LIBS="$ALSA_LIBS -lasound -lm $LIBDL $LIBPTHREAD"
 LIBS="$ALSA_LIBS $LIBS"
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ALSA_LIBS" >&5
 $as_echo "$ALSA_LIBS" >&6; }
@@ -8366,13 +8366,13 @@ else
   as_fn_error $? "Error: Need sqrtf" "$LINENO" 5
 fi
 
-  { $as_echo "$as_me:${as_lineno-$LINENO}: checking for pthread_create in -lpthread" >&5
-$as_echo_n "checking for pthread_create in -lpthread... " >&6; }
+  { $as_echo "$as_me:${as_lineno-$LINENO}: checking for pthread_create in $LIBPTHREAD" >&5
+$as_echo_n "checking for pthread_create in $LIBPTHREAD... " >&6; }
 if ${ac_cv_lib_pthread_pthread_create+:} false; then :
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lpthread  $LIBS"
+LIBS="$LIBPTHREAD  $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
@@ -8407,7 +8407,7 @@ if test "x$ac_cv_lib_pthread_pthread_create" = xyes; then :
 #define HAVE_LIBPTHREAD 1
 _ACEOF
 
-  LIBS="-lpthread $LIBS"
+  LIBS="$LIBPTHREAD $LIBS"
 
 else
   as_fn_error $? "Error: need PTHREAD library" "$LINENO" 5
